staff/assets/orgacerts.sty

```diff
@@ -35,27 +28,33 @@
 \newcommand{\cert}[2]{
     \seitenkopf
     
-    \vspace{1cm}
+    \vspace{2cm}
 
     \begin{center}
       \huge
-      Zeugnis über die Organisation der Orientierungsphase \\
+      Bescheinigung über Mitarbeit bei der Orientierungsphase \\
       vom \ophasenTermin
     \end{center}
 
-    \vspace{4cm}
-    
-    \noindent Der/die Student/in der Informatik \\
+    \vspace{0.7cm}
+    \center \includegraphics[height=5cm]{\wesenBild}
+    \vspace{1.2cm}
     
-    \hspace{1.5cm} \textbf{#1 #2} \\
-    
-    \noindent hat im \semesterName \  die \\
-    
-    \hspace{1.5cm} \textbf{Orientierungsphase der Fachschaft Informatik} \\
-    
-    \noindent mit Erfolg organisiert.
+    \begin{center}
+        \Large Wir danken\\
+        \vspace{0.7cm}
+        \Huge \textbf{#1#2}\\
+        \vspace{0.7cm}
+        \Large für tatkräftige Hilfe bei der Organisation der Orientierungsphase\\
+        am Fachbereich Informatik der TU Darmstadt.
+    \end{center}
+
+    \vspace{3.5cm}
+
+    \begin{tabularx}{15cm}{cp{3cm}c}
+    Prof. Dr. rer. nat. Max Mühlhäuser & & Prof. Dr.-Ing. Michael Goesele\\
```

Comment:
in the latest design, this table is specifically `\hspace`-adjusted. The shift value depends on the rendered text length of the names. Therefore I will leave the names hard-coded for now.